### ISSUE TITLE
Accessibility: Popovers have no focus after opening

### ISSUE THREAD

**Issue by sedlatschek** [laravel/nova-issues#3393]
- Laravel Version: 8.41.0
- Nova Version: 3.25.0
- PHP Version: 7.4.6
- Database Driver & Version: MySQL 5.7
- Operating System and Version Win10
- Browser type and version: Chrome 88 / Edge 90

### Description:

When a dropdown-trigger is triggered by keyboard, the focus is not set to the resulting popover. This makes it hard for blind users to interact with the popover. The user has to tab through the whole document to get to the popover. It is impossible for a blind user to tell whether the popover has opened and how to reach it.

This seems to be a general problem for all popovers, as it is present for the `select-all`, `lenses` and `filter-selector` dropdowns.

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. Open any resource index view
2. Tab trough to the filter dropdown-trigger 
![filter dropdown trigger](https://user-images.githubusercontent.com/58665536/118529651-0b4a2600-b744-11eb-9a27-095866d375fa.png)
3. Press `Space` on the keyboard
4. Try to focus the popover by only using the keyboard

**Comment by davidhemphill** [comment]
👋 Hi there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.
